.github/workflows/publish-typist-package-release.yml

```diff
@@ -13,8 +13,11 @@ permissions:
     # Enable the use of OIDC for npm provenance
     id-token: write
     # Enable the use of GitHub Packages registry
-    contents: read
     packages: write
+    # Enable `semantic-release` to publish a GitHub release and post comments on issues/PRs
+    contents: write
+    issues: write
+    pull-requests: write
```

Comment:
These changes are unrelated to this PR, and `semantic-release` was fully working before without these permissions, however, the documentation states that these are the required permissions for every feature to work, so I'm adding them for good measure.